### ISSUE TITLE
No special sounds for slabs and stairs

### ISSUE THREAD

**Issue by Wuzzy2** [starninjas/sounds_redone#1]
The special sounds for stone bricks, brick blocks, etc. do not work for slabs and stairs.

**Comment by starninjas** [comment]
Yeah, I've noticed that, it's something I'll look into. Not 100% sure how I'll fix it.
